src/coreclr/src/build.proj

```diff
@@ -10,6 +10,7 @@
     <ProjectReference Condition="'$(BuildManagedTools)' == 'true'" Include="tools/runincontext/runincontext.csproj" />
     <ProjectReference Condition="'$(BuildManagedTools)' == 'true'" Include="tools/r2rdump/R2RDump.csproj" />
     <ProjectReference Condition="'$(BuildManagedTools)' == 'true'" Include="tools/ReadyToRun.SuperIlc/ReadyToRun.SuperIlc.csproj" />
+    <ProjectReference Condition="'$(BuildManagedTools)' == 'true'" Include="tools/ILVerification/ILVerification.csproj" />
```

Comment:
Does this really have to land in coreclr. I'd love to use this even with coreclr build disabled

Reply:
90+% of sources for ILVerify are shared with the CoreCLR AOT compiler. It gets unnatural to have the sources in one place and the project including the sources in different place, just to make it fit better with top level build script.

Do any of these options look reasonable to you?
- Use it from package if you do not want to build CoreCLR?
- Invoke the build of this .csproj file from Mono build by default?
- Add a new subset for this so that you can build just the tool, without the rest of CoreCLR?

This is touching on larger problem that the current coreclr/mono directory structure is unfriendly to sharing. It is reasonable expect that we will want to share more and more in future. What should the directory structure friendly to sharing between coreclr/mono look like?

Reply:
> Do any of these options look reasonable to you?

I think I'd like to use `dotnet tool ilverifier` whether I'd building with coreclr or not.

> This is touching on larger problem that the current coreclr/mono directory structure is unfriendly to sharing.

I think this is not necessarily about runtime sharing but about having a way to build shared tools for dotnet/runtime but I'm not opposed to solving both of these at once.